docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.5'
+
+services:
+
+  vegeta:
+    build: .
+    image: flaviostutz/vegeta
+    ports:
+      - 8880:8880
+    environment: 
+      - TARGET_URL=https://www.uol.com.br
+      - DURATION=60s
+      - REQUESTS_PER_SECOND=5
+
+  prometheus:
+    image: flaviostutz/prometheus
```

Comment:
Same for this image.

Reply:
This is an extension of the official prometheus image. In this image I added the option to pass scrape configurations using environment variables. In official prometheus image, they place configuration files in volumes, which is not a good pattern because it generates a "stateful" situation on operation (one has to create a configuration file and place in volumes, oposed to simply setting configurations in ENV parameters).

If you like we can remove this service from docker-compose.yml.